--- a/sound/soc/amd/yc/acp6x-mach.c	2025-06-04 18:15:11.000000000 +0530
+++ b/sound/soc/amd/yc/acp6x-mach.c	2025-06-16 19:48:46.241542820 +0530
@@ -507,6 +507,15 @@
 			DMI_MATCH(DMI_BOARD_NAME, "8A42"),
 		}
 	},
+	
+	{
+		.driver_data = &acp6x_card,
+		.matches = {
+			DMI_MATCH(DMI_BOARD_VENDOR,"HP"),
+			DMI_MATCH(DMI_PRODUCT_NAME,"Victus by HP Gaming Laptop 15-fb1xxx"),
+		}
+	},
+
 	{
 		.driver_data = &acp6x_card,
 		.matches = {
